internal/common/utils.go

```diff
@@ -211,38 +209,6 @@ func CompareStrStrMap(a map[string]string, b map[string]string) bool {
 	return true
 }
 
-func MakeAddressable(name string, addr *models.Addressable) (*models.Addressable, error) {
```

Comment:
Same as const, and whitespace changes. At minimum this belongs in its own commit ("rm unused Addressable code"), or separate PR.

Reply:
will do